README.md

```diff
@@ -1 +1,292 @@
+
+* `process`: Contains the information about the system and environment, e.g., node version (`process.versions.node`)
+* `module`: Exports the functionality to be used by other programs/files, i.e., utilized to create modules
+* `exports`: Exports the functionality to be used by other programs. Alias of `module.exports`.
+* `require`: Imports the functionality from another file/module/package, i.e., utilizes modules
+* `Buffer`: Binary data type 
+* `__dirname`: Path and folder name of the current script
+* `_filename`: Filename of the current script 
+
+Know that `global` object has interfaces specific to Node. Others are the same as in browser JavaScript:
+
+* `setInterval()`
+* `setTimeout()`
+* `console`
+* `clearTimeout(t)`
+* `clearInterval(t)`
+
+Note: Developers you can access them with or without referring to the `global` object. For example, both `global.process` and `process` work the same. 
+
+Process, modules and Buffer requires their own lessons, so we'll cover them one-by-one later. For now let's look more into `process`. 
+
+## Explicit and Implicit Global Scoping
+
+Let's talk about scoping for a moment. We can attach any arbitrary (meaning we come up with a name) property to a `global` object explicitly, e.g., 
+
+```js
+global.name = 'React Quickly'
+console.log(name) // React Quickly
+```
+
+This is explicit global scoping because we wrote `global.name`. This variable become available not only in the current script, but in all other scripts which this instance uses. For example you can set the global name variable in the main script and it will be accessible in imported modules. More on modules later. Let's explore implicit global scoping.
+
+When it comes to scoping, browser JavaScript was notorious for it's "buggy" behavior of leaking variables into a global scope. For example, if `var _user = {admin: false}` code is run in DevTools `var user` will create a global object `window._user`. 
+
+Note: If we don't use `var` the variable will be a `window` as well. 
+
+So if you run this code in the DevTools console:
+
+```js
+user = {admin: false}
+var _user = {admin: false}
+
+console.log(_user, window._user, window._user)
+console.log(user, window.user, window.user)
+```
+
+You would get these results which illustrate that global window references were created:
+
+```
+Object {admin: false} Object {admin: false} Object {admin: false}
+Object {admin: false} Object {admin: false} Object {admin: false}
+```
+
+Now let's replace `window` with global for Node. What the result of this snippet would be if you run it as a separate file (`$ node main.js`)?
+
+```js
+user = {admin: false}
+var _user = {admin: false}
+
+console.log(_user, global._user, GLOBAL._user)
+console.log(user, global.user, GLOBAL.user)
+```
+
+The result will signify that the `var _user` did not create a global reference, only a local one.
```

Comment:
can you break the console.log up a little bit? this result is slightly unclear.

`console.log(_user)` returns `{ admin: false}`
`console.log(global.user)` returns `undefined`
`console.log(GLOBAL._user)` returns `undefined`

BUT 
`console.log(user)` returns `{ admin: false}`
`console.log(global.user)` returns `{ admin: false}`
`console.log(GLOBAL.user)` returns `{ admin: false}`

I'm also not totally sure i understand why `var _user` didn't  create a global object